embedded-graphics/src/image/image16bpp.rs

```diff
@@ -0,0 +1,208 @@
+//! 16 bits per pixel images. Every two bytes define the color for each pixel.
+//! 
+//! You can convert an image to 16BPP for inclusion with `include_bytes!()` doing the following
+//! 
+//! Converting the Image to 16bit BMP (GIMP can do this), then running 
+//! ```bash
+//! tail -c $bytes image.bmp > image.raw
```

Comment:
Can ImageMagick do this? It would be great to have a copyable command to allow users to convert multiple image formats into 16BPP data easily.

Reply:
I found a way to do it in ImageMagick, but the result seems slightly washed out compared to the GIMP method, so I've left both methods in.

Reply:
Thanks for investigating. I'll leave it as is :)